### PR TITLE
added new Js Question

### PR DESCRIPTION
**Pull request by Viratsoam**

### CODE REVIEW

File: javascript/index.html [hunk 1]
```diff
@@ -72,13 +72,6 @@ <h4>
 
                 </a>
             </li>
-            <li>
-                <a href="./answers/closure1.html">
```
Comment (arpansac) [inline javascript/index.html]: Removed this?

Please fix, take a fresh pull and send a fresh PR!

File: javascript/answers/q4.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What will be the output of the code below?">
```
Comment (arpansac) [inline javascript/answers/q4.html]: Please change this to the text of the question (content="")

**Comment by dakshkhetan** [comment]
Any updates on this? This PR will be closed in 2 days.

**Comment by dakshkhetan** [comment]
@Viratsoam Question 6 has already been added by someone. Please take a fresh pull (upstream master) and resolve conflict. :)

**Review by arpansac — APPROVED** [review]

**Pull request merged**